msteams-platform/bots/what-are-bots.md

```diff
@@ -59,6 +62,9 @@ You can use the Teams bot APIs to get information for members of a chat or team.
 |Sample name | Description | C# | Node.js |
 |----------------|-----------------|--------------|--------------|
 | Bot daily task reminder| Demonstrate how to schedule a recurring task and get a reminder at a scheduled time. | [View](https://github.com/OfficeDev/Microsoft-Teams-Samples/tree/main/samples/bot-daily-task-reminder/csharp) | [View](https://github.com/OfficeDev/Microsoft-Teams-Samples/tree/main/samples/bot-daily-task-reminder/nodejs) |
+| Hello World Bot | This is a simple hello world application with both Bot and Message extension capabilities. | [View](https://github.com/OfficeDev/TeamsFx-Samples/tree/v1.0.0/hello-world-bot) |
```

Comment:
The links in the table Code Samples should belong to Column Node.js rather than C#.